test/tests/functional/pbs_mom_dynamic_resource_ext.py

```diff
@@ -316,11 +314,11 @@ def test_t11(self):
         try:
             self.server.manager(MGR_CMD_DELETE, RSC, id='foo')
         except PbsManagerError as e:
-            self.assertTrue("Resource busy on server")
+            self.logger.info(e.message)
```

Comment:
why did you remove the assertion? I think you need to update it to something
assertTrue("Resource busy on server" in e.msg[0])